src/lib.rs

```diff
@@ -198,27 +196,29 @@ impl Default for CapabilitiesContext {
             blob: Box::new(DefaultObjectStore::new()),
             extras: Box::new(DefaultExtras::new()),
             events: Box::new(DefaultEventStreams::new()),
-            log: Box::new(DefaultLogger::new()),
+            log: Box::new(AutomaticLogger::new()),
         }
     }
 }
 
-static LOGGER: AutomaticLogger = AutomaticLogger{l: DefaultLogger{}};
-
+static LOGGER: AutomaticLogger = AutomaticLogger {};
 
 impl CapabilitiesContext {
     /// Creates a new capabilities context. This is invoked by the `actor_receive` macro
-    pub fn new() ->  CapabilitiesContext {
-
-    log::set_logger(&LOGGER).map(|()| log::set_max_level(LevelFilter::Trace));
+    pub fn new() -> CapabilitiesContext {
+        match log::set_logger(&LOGGER) {
+            std::result::Result::Ok(_) => console_log("Set Logger: OK"),
```

Comment:
This may still be a little misleading. It's expected that second (and third, etc) calls to set_logger will fail. We don't want to give the impression that something is wrong when it's really BAU

Reply:
ok ill clean this up